Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -14,6 +14,17 @@
 
 import Foundation
 
+extension String {
+    func leftPadding(toLength: Int, withPad: String) -> String {
```

Comment:
Could you please add some tests for this function?

Reply:
@saeta Thank you for the review. I'll work to add some tests.

Reply:
@saeta 
When I'll add some tests, which do you prefer to make a new test file or edit [BlackHoleReporter.swift](https://github.com/google/swift-benchmark/blob/master/Tests/BenchmarkTests/BlackHoleReporter.swift)?

Reply:
Done.